### ISSUE TITLE
Hide comments in visual mode

### ISSUE THREAD

**Issue by 0nko** [wordpress-mobile/AztecEditor-Android#541]
Add a configuration option to hide the HTML comments in visual mode (in preparation for Gutenberg).

**Comment by maxme** [comment]
There might be another change to consider to support basic Gutenberg blocks: images are wrapped in a `figure` tag like:

```
<!-- wp:core/image {"id":316} -->
<figure class="wp-block-image"><img src="https://bia.is/hola-que-tal.jpg" alt="" /></figure>
<!-- /wp:core/image -->
```

Aztec could simply ignore visually these tags as a first step.

**Comment by daniloercoli** [comment]
Fixed in #592
